Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/SortAdapter.cpp

```diff
@@ -21,7 +22,7 @@ namespace AZ::DocumentPropertyEditor
             m_sortActive = active;
             if (active)
             {
-                // TODO: This method should genenerate patches to/from source once custom patching is implemented
+                // TODO: This method should generate patches to/from source once custom patching is implemented
```

Comment:
suggestion: While we're here is it worth updating this TODO to a GHI if this issue is being tracked somewhere?